packages/itwinui-react/src/core/Tabs/Tabs.tsx

```diff
@@ -208,6 +208,11 @@ export const Tabs = (props: TabsProps) => {
     }
   }, [focusedIndex]);
 
+  const [isAnimated, setIsAnimated] = React.useState(false);
+  React.useEffect(() => {
+    setIsAnimated(true);
+  }, []);
```

Comment:
This can actually be a generic hook called `useIsClient`.

```js
const useIsClient = () => {
  const [isClient, setIsClient] = React.useState(false);

  React.useEffect(() => {
    setIsClient(true);
  }, []);

  return isClient;
};
```

you can add it under `utils/hooks/` and then use it like this:
```js
const isClient = useIsClient();
```

Reply:
Interesting. I have added this hook into util and use it in Tabs based on your suggestion. I have also added the nextjs playground code for testing. In this playground, I don't see the bug anymore. But I still see it in Storybook I feel like. It's so subtle so really hard to tell O_O

Reply:
I left the playground code too for reviewers to test. But will try to remember to remove it 🤔

Reply:
remove it and add changeset. otherwise this PR looks good

Reply:
Thanks. Done.